core/math/transform_3d.cpp

```diff
@@ -71,11 +71,17 @@ void Transform3D::rotate_basis(const Vector3 &p_axis, real_t p_angle) {
 
 Transform3D Transform3D::looking_at(const Vector3 &p_target, const Vector3 &p_up) const {
 	Transform3D t = *this;
+#ifdef MATH_CHECKS
+	ERR_FAIL_COND_V_MSG(origin.is_equal_approx(p_target), t, "The transform's origin and target can't be equal.");
```

Comment:
In this case I returned the current transform, not sure if it would make more sense to return an empty transform.

Reply:
Consistency should be preferred, so either return an empty transform, or change `Basis::looking_at()` to be a method and return the current basis from that aswell.
The former would be preferred.

Reply:
If a user tells a node3d to look at a position every frame, if the positions were equal it would reset the transform. Hmm. I do agree consistency is better, but I'll keep this discussion open.